src/figuresClasses.ts

```diff
@@ -1,11 +1,98 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string;
+
+  color: string;
```

Comment:
[CODE STYLE] - The `shape` property in the `Triangle` class should have a literal type 'triangle' instead of a generic string, as the shape is not meant to change. Consider using the literal type to enforce this constraint.